app/renderer/components/Session/ServerTabMobitru.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Form, Input, Row, Col } from 'antd';
+import { INPUT } from '../AntdTypes';
+
+const mobitrWebDriverUrlPlaceholder = (t) => {
+  if (process.env.MOBITRU_WEBDRIVER_URL) {
+    return t('usingDataFoundIn', {environmentVariable: 'MOBITRU_WEBDRIVER_URL'});
+  }
+  return 'https://app.mobitru.com/wd/hub';
+};
+
+const mobitruBillingUnitPlaceholder = (t) => {
+  if (process.env.MOBITRU_BILLING_UNIT) {
+    return t('usingDataFoundIn', {environmentVariable: 'MOBITRU_BILLING_UNIT'});
+  }
+  return 'personal';
+};
+
+const mobitruAccessKeyPlaceholder = (t) => {
+  if (process.env.MOBUTRU_ACCESS_KEY) {
```

Comment:
I assume this typo is not intentional? :)

Reply:
Fixed. Thank you!